src/app/parcels/ActionBar/ActionModals/ShoppingListModal.tsx

```diff
@@ -73,6 +73,9 @@ const getPdfErrorMessage = (error: ShoppingListPdfError): string => {
         case "invalidFamilySize":
             errorMessage = "Invalid family size for shopping list PDF.";
             break;
+        case "inactiveClient":
```

Comment:
We could do it like this. Ricky suggested the approach I did, but I think it's only strength is its less effort. I'm happy to do it this way if you want, but this will be quite a bit of extra work as we'll have both the case where only parcels with inactive clients are selected, and the case where a mixture of parcels with active and inactive clients are selected. what we display would have to be different. Happy to spend the time on it if you want me to, lmk :)

Reply:
We had the same debate about the Day Overview pdf on another PR - I agree it's less effort and would get flagged sooner, but it might also inconvenience people for and undefined amount of time. But you're right this PR is already big enough - maybe raise a ticket to change it later?

Reply:
Ticket raised [VFB-231](https://softwiretech.atlassian.net/browse/VFB-231)